src/compat_tests.rs

```diff
@@ -183,8 +192,16 @@ mod pgp {
             )?
             .epoch(1)
             .pre_install_script("echo preinst")
-            .add_changelog_entry("me", "was awesome, eh?", 123_123_123)
-            .add_changelog_entry("you", "yeah, it was", 12_312_312)
+            .add_changelog_entry(
+                "me",
+                "was awesome, eh?",
+                chrono::Utc.timestamp_millis_opt(1681411811).unwrap(),
```

Comment:
I think you want `timestamp_opt()` not `timestamp_millis_opt()`

Reply:
Good catch!